server.ts

```diff
@@ -614,6 +618,17 @@ app.post(
 );
 app.post("/api/v1/test/:user_id/send_email", usersSendEmails.sendEmailTest);
 
+// Center expenses paid
+app.get("/api/v1/centers/:center_id/expense/paid", centerExpensesPaid.getCenterExpensePaid)
```

Comment:
Let's name all expenses endpoints in plural (e.g. expenses/paid instead of expense/paid).